docs/GettingStarted.md

```diff
@@ -243,7 +243,7 @@ scanner on the `dependencies` scope in this tutorial. If you also want to scan t
 advised to configure a cache for the scan results as documented in the README to speed up repeated scans.
 
 ```bash
-$ cli/build/install/ort/bin/ort scan -d [analyzer-output-path]/all-dependencies.yml -o [scanner-output-path] --scopes dependencies
+$ cli/build/install/ort/bin/ort scan --analyzer-result-file [analyzer-output-path]/all-dependencies.yml -o [scanner-output-path] --scopes dependencies
```

Comment:
While at it, could you please also change "all-dependencies.yml" to "analyzer-result.yml", which has changed in 4c7040dc9586ddc6cc08d8831bd0980fdbc6db7c? Feel free to do this in a new commit, or amend this one.

Reply:
Sure, amended this change accordingly.